fix(guides): update misleading copy in Netlify deploy guide (#8952)
withastro/docs: src/content/docs/en/guides/deploy/netlify.mdx

### Adapter for SSR

> To enable SSR in your Astro project and deploy on Netlify, including using Netlify's edge functions:

Author: I also removed this, because it seemed too vague and possibly misleading

> To enable SSR in your Astro project and deploy on Netlify:
> 
> Add [the Netlify adapter](/en/guides/integrations-guide/netlify/) to enable SSR in your Astro project with the following `astro add` command. This will install the adapter and make the appropriate changes to your `astro.config.mjs` file in one step.

Reviewer: I actually think we can probably collapse these two paragraphs as they basically repeat each other:

```suggestion
Add [the Netlify adapter](/en/guides/integrations-guide/netlify/) to enable SSR in your Astro project and deploy to Netlify with the following `astro add` command. This will install the adapter and make the appropriate changes to your `astro.config.mjs` file in one step.
```

    });
    ```

>     You can also deploy your project's Astro Middleware using Netlify's Edge Functions by adding `edgeMiddleware: true` to the Netlify adapter's configuration:

Reviewer: Nit, but we usually don’t uppercase Astro features:

```suggestion
    You can also deploy your project's Astro middleware using Netlify's Edge Functions by adding `edgeMiddleware: true` to the Netlify adapter's configuration:
```

        }),
      });
    ```

>     If you use Middleware to implement authentication, redirects, or similar things, you may need to enable this flag. Otherwise, your Middleware will not run at runtime for pre-rendered pages.

Reviewer: Same here re: capitalisation, but also suggesting flipping the wording for a more positive “do X to achieve Y” pattern rather than “do X to avoid Z”.

```suggestion
    
    Set `edgeMiddleware: true` if you want to run middleware for pre-rendered pages. This allows you to use middleware to implement authentication, redirects, or similar things, while still using static HTML output.
```

Reviewer 2: Great call, Chris!

For your consideration, I'll also throw this slight tweak out there to frontload the "to do X" part:

> To run middleware for pre-rendered pages, set `edgeMiddleware: true`. ...

Reviewer: Ah, great call, yes. I honestly initially aimed for that and didn’t quite land on a formulation I liked, but this works great. Will update my suggestion to match!
